src/com/ableton/VirtualEnv.groovy

```diff
@@ -25,6 +27,11 @@ class VirtualEnv implements Serializable {
   }
 
 
+  def create() {
```

Comment:
I am against this change. However, I can propose the following alternative: make `create` a `static` method which does
```
  venv = new VirtualEnv(script, python)
  venv.script.sh("virtualenv --python=${python} ${venv.destDir}")
  return venv
```
and then call `VirtualEnv.create(script, python)` in the singleton `create`.

Reply:
Ok, I could live with that.

Reply:
Do you intend to address my comment?

Reply:
I thought I did? I made the `create` method in `VirtualEnv` static, and fixed the tests accordingly. Is there some miscommunication going on here?

Reply:
Sorry. I didn't realize this was the only commit that was re-written.